tests/test_replace_func.py

```diff
@@ -0,0 +1,174 @@
+import os
```

Comment:
[optional] It could be nice if we can add a real-world example as a testcase and test it by onnxruntime. It would be a great example for users to see the usage of this feature. For example, is it possible to export `np.full` by `ConstantOfShape`? I'm not sure, if it's easy - it takes a shape as an input and IIUC shape related stuff does not work well with FunctionNode-based ONNX-chainer.

Reply:
I tried adding `np.full` converter test using `as_funcnode` deco.

Reply:
Nice, thanks!